docs/outbound-transfers/Basics.md

```diff
@@ -0,0 +1,62 @@
+---
+tags: [Outbound Transfers]
+---
+
+# Outbound transfer basics
+
+<!-- theme: warning -->
+> Outbound transfers are currently in beta, and are only available in sandbox.
+
+## What is an outbound transfer?
+
+An outbound transfer is a transfer of an existing account currently managed by WealthKernel (on your behalf) to an external provider. This can be initiated in a couple of ways:
```

Comment:
Is is worth mentioning the account types we can transfer out, or is it already expected that any type can be transferred?

Reply:
It should be possible to transfer any account out already, yeah